### PR TITLE
Fix: One table only has one tab and show warning dialogs when closing unsaved tabs

### LINKED ISSUE #1700: BUG: Closing all tabs does not give warning of unsaved changes
**To Reproduce**
1. Make some changes in many tabs (writing queries, creating a table, etc.)
2. Right click the tab and `Close All`/`Close Other`/`Close Tabs to The Right`

**Expected behavior**
It should give a warning like when closing a single tab.

**Additional context**
It should be able to handle unsaved changes in multiple tabs too.

### PR DESCRIPTION
**Pull request by azmy60**
closes #1700

### CODE REVIEW

File: apps/studio/src/components/CoreTabs.vue [hunk 1]
```diff
@@ -635,7 +613,15 @@ export default Vue.extend({
         }
       }
     },
-    async close(tab) {
+    async close(tab: OpenTab) {
+      if (tab.unsavedchanges) {
+        const confirmed = await this.$confirm(
+          `really close [icon here] ${tab.title}?`,
```
Comment (rathboma) [inline apps/studio/src/components/CoreTabs.vue:619]: looks like this has a placeholder? `[icon here]`?

**Review by rathboma — CHANGES_REQUESTED** [review]
looks good, but seems like there's some placeholder text?

File: apps/studio/src/components/CoreTabs.vue (hunk 1 above)
Comment (azmy60) [reply apps/studio/src/components/CoreTabs.vue:619]: Lol, I forgot. My bad.

Comment (rathboma) [reply apps/studio/src/components/CoreTabs.vue:619]: Once this is done, I'll get it merged it. Probably for the next update

Comment (azmy60) [reply apps/studio/src/components/CoreTabs.vue]: image

done :)

**Review by rathboma — APPROVED** [review]

**Pull request merged**